packages/form/src/components/SelectInputFieldV2/__stories__/Multiple.stories.tsx

```diff
@@ -5,4 +5,14 @@ import { planets } from './resources'
 
 export const Multiple: StoryFn<
   ComponentProps<typeof SelectInputFieldV2>
-> = () => <SelectInputFieldV2 name="options" multiselect options={planets} />
+> = () => (
+  <SelectInputFieldV2
+    selectAllGroup
+    selectAll={{
+      label: 'Select all',
+    }}
+    name="options"
+    multiselect
+    options={planets}
+  />
+)
```

Comment:
There is already a story for select all `packages/ui/src/components/SelectInputV2/__stories__/SelectAll.stories.tsx`

Reply:
ok my bad it's the field